build/Dockerfile.operator

```diff
@@ -0,0 +1,14 @@
+FROM centos:8
+
+RUN dnf install -b -y dnf-plugins-core && \
```

Comment:
None of these are needed. Let's just use minimal container and our built binary.

Reply:
Either let's build tiny separate image for the operator or just keep the binary as a part of the already existing image. It may help us get some basics in. We can split it later (that will be probably done on OpenShift anyway)